spec/features/hide_sign_up_from_cnx_users_spec.rb

```diff
@@ -1,12 +1,15 @@
 require 'rails_helper'
 
+# TODO KC - I think you would have written this way back -- any recollection
+# of what this spec is trying to show?  I have temporarily disabled the scenario
```

Comment:
This is the original test: https://github.com/openstax/accounts/commit/cb738c341bf0d7d633f575964eeb2de2a3c06b62

This commit removed most of it: https://github.com/openstax/accounts/commit/69ddd25c8a88cfce6bccdb5faeac6863199b672b

I think it's safe to say that we don't need this test anymore.